[DOCS-11792] add de-duplication setting to SNMP docs (#31076)
DataDog/documentation: content/en/network_monitoring/devices/snmp_metrics.md

2. Edit the [`datadog.yaml`][8] Agent configuration file to include all the subnets for Datadog to scan. The following sample config provides required parameters, default values, and examples for Autodiscovery.

> 3. Optionally, enable [de-duplication][11] of devices during Autodiscovery of the Agent (disabled by default). Requires Agent version `7.67` or higher.

Reviewer: ```suggestion
3. Optionally, enable [de-duplication][11] of devices during Autodiscovery of the Agent. This feature is disabled by default and requires Agent version `7.67+`.
```
I think this confused me on which is disabled by default, the deduplication or the Autodiscovery of the Agent. Does this edit keep the same meaning?

Author: oh I see what you mean yes good catch! I like your suggestion
